unassigner/prepare_strain_data.py

```diff
@@ -94,4 +96,6 @@ def download_type_strain_data(output_dir=None, metadata_fp=None, seqs_fp=None):
         output_dir = os.getcwd()
     metadata_fp = use_or_download(metadata_fp, LTP_METADATA_URL, output_dir)
     seqs_fp = use_or_download(seqs_fp, LTP_SEQS_URL, output_dir)
-    return process_ltp_seqs(seqs_fp, output_dir)
+    ltp_fp = process_ltp_seqs(seqs_fp, output_dir)
+
+    return metadata_fp, seqs_fp, ltp_fp
```

Comment:
You only need `ltp_fp` when you call the function. Seems like `metadata_fp` and `seqs_fp` don't need to be added here.